ckanext/iati/theme/templates/home/index.html

```diff
@@ -7,7 +7,7 @@
     {{ self.flash() }}
 
     {% trans %}
-      <h1><strong>IATI Data</strong> is the place to search, register and use IATI data.</h1>
+      <h1><strong>IATI Registry</strong> is the place register your organisation and IATI datasets.</h1>
```

Comment:
> is the place register your organisation

looks like this is missing the word “to”

Reply:
In fact, I wonder if this should also say “**The** IATI Registry is the place **to** register […]”

Reply:
@amy-silcock do you agree with @andylolz 's suggestion?